BUILD.rst

```diff
@@ -22,7 +22,7 @@ their dependencies. You can install these packages using the standard
 - pkg-config
 - make
 - mingw64-x86_64-gcc-core
-- mingw64-x86_64-g++
+- mingw64-x86_64-gcc-g++
```

Comment:
As per cygwin package list ( https://cygwin.com/packages/package_list.html#m) this change is correct but why are we requiring c++/g++?  Could anyone using cygwin confirm we need this, and `mingw64-x86_64-gcc-core` is not enough?

Anyway, this chunk is unrelated and should be a separate commit or PR.

Reply:
I can check it tomorrow

Reply:
I removed the g++ package and the project rebuilt successfully.

Reply:
Thanks for the follow up.
Yes, please cleanup the first commit (i.e., remove the editing of Build.rst from it) and force push. Then we can merge this.

Reply:
I confirm, build is ok without g++ installed